templates/gocd-server/0/docker-compose.yml

```diff
@@ -1,4 +1,6 @@
 gocd-server:
+  ports:
+  - 8153:8153/tcp
```

Comment:
Can you make this a question in the rancher-compose.yml. Hard coded ports cause issues.